MMM-COVID19.css

```diff
@@ -22,6 +22,10 @@
   color: red;
 }
 
+.codid td.serious {
+  color: red;
```

Comment:
please choose another color for `serious cases`. I don't want to have them read as `deaths`